src/consent-manager-builder/index.tsx

```diff
@@ -246,7 +249,8 @@ export default class ConsentManagerBuilder extends Component<Props, State> {
       preferences,
       isConsentRequired,
       destinationPreferences,
-      workspaceAddedNewDestinations: Boolean(workspaceAddedNewDestinations)
+      workspaceAddedNewDestinations: Boolean(workspaceAddedNewDestinations),
+      havePreferencesChanged: initialPrefencesHaveValue
```

Comment:
We don't need this here. If we set this to true, if will impact the `handleSaveConsent` and leads to unnecessary refresh.